README.md

```diff
@@ -191,6 +221,34 @@ If you'd like to create a note in an MR (like a comment, but not linked to a spe
 require("gitlab").create_note()
 ```
 
+### Discussions signs and diagnostics
+
+By default when reviewing files you will see signs and diagnostics ( if enabled in configuration ). When cursor is on diagnostic line you can view discussion thread by using `vim.diagnostic.show`. You can also jump to discussion tree where you can reply, edit or delete discussion.
+
+```lua
+require("gitlab").move_to_discussion_tree_from_diagnostic()
+```
+
+The `discussion_sign` configuration controls the display of signs for discussions. The `enabled` option turns on/off the signs. `text` sets the sign text. `linehl`, `texthl`, `culhl`, and `numhl` customize the line highlighting, text highlighting, column highlighting, and number highlighting respectively. Keep in mind that these can be overridden by other configuration (for example diffview.nvim highlights). `priority` controls the sign priority order (when multiple signs are placed on the same line, the sign with highest priority is used). The `helper_signs` table configures additional signs for multiline discussions in order to show the whole context. `enabled` turns on/off the helper signs. `start`, `mid`, and `end` set the helper sign text.
+
+The `discussion_diagnostic` configuration customizes the diagnostic display for discussions. The `enabled` option turns on/off the diagnostics. `severity` sets the diagnostic severity level and should be set to one of `vim.diagnostic.severity.ERROR`, `vim.diagnostic.severity.WARN`, or `vim.diagnostic.severity.INFO`, `vim.diagnostic.severity.HINT`. `code` specifies a diagnostic code. `display_opts` configures the diagnostic display options where you can configure values like:
+
+- `virtual_text` - Show virtual text for diagnostics.
```

Comment:
Perhaps I'm doing something wrong, but these options don't seem to be applying for me... The only one that's working is setting `virtual_text` to a boolean value.

Reply:
I tried both `{virtual_text = false, underline = false}` and these works for me -> these were main reason for having `display_opts` to be able to have different settings than regular diagnostics.

I think it makes sense to remove `float` because that is basically configuration for floating window ( table ) and probably too confusing here (`:h vim.diagnostic.open_float`). Same goes for `severity_sort` because if works but only within our defined namespace so I understand the confusion here. Advanced users will be still able to put there any settings from `:h vim.diagnostic.config`

Reply:
but the thing with float is interesting because this respect settings
```
:lua vim.diagnostic.open_float({header = "Hi",       source = false,        border = "rounded",})
```
but my shortcut does not :thinking:

Reply:
Yeah that's what I was noticing, I was hoping to be able to set specific float options for Gitlab (e.g. title, formatting the text, etc) but they weren't working for me.

Reply:
It seems that I cannot make the float work. One thing is that I have key binding with specific settings which should override anything:
```
    "n",
    "<leader>cd",
    "<cmd>lua vim.diagnostic.open_float(0, {scope='line', border = 'rounded'})<CR>",
    { silent = true }
```

but even when I use `open_float` without config it does not respect settings. I can try to dig into that in following days :thinking:

Reply:
That's fine, let's not worry about it as part of this MR. Perhaps worth asking about in other forums.